templates/geosafe/analysis/create.html

```diff
@@ -509,6 +511,7 @@
         var aggregation_layer_cbo = $("#id_aggregation_layer");
         var impact_function_cbo = $("#id_impact_function_id");
         var last_impact_id;
+        sliderVal = 8;
```

Comment:
Use underscore case for variable name. We tried use the same convention with geonode.
change sliderVal to slider_val or slider_value